Sources/Parsing/Documentation.docc/Articles/Parsers/Int.md

```diff
@@ -42,36 +42,39 @@ UTF-8 code units (see <doc:StringAbstractions> for more info).
 
 Typically Swift can choose the correct overload by using type inference based on what other parsers
 you are combining `Int.parser()` with. For example, if you use `Int.parser()` with a
-`Substring` parser, say the literal "," parser (see <doc:String> for more information), Swift
+`Substring` parser, say the literal `","` parser (see <doc:String> for more information), Swift
 will choose the overload that works on substrings:
 
 ```swift
-try Parse {
+let parser = Parse {
   Int.parser()
   ","
   Int.parser()
 }
-.parse("123,456") // (123, 456)
+
+try parser.parse("123,456") // (123, 456)
 ```
 
 On the other hand, if `Int.parser()` is used in a context where the input type cannot be inferred,
 then you will get an compiler error:
 
 ```swift
-try Parse {
-  Int.parser()
-  Bool.parser() // 🛑 Ambiguous use of 'parser(of:)'
+let parser = Parse {
+  Int.parser() // 🛑 Ambiguous use of 'parser(of:isSigned:radix:)'
+  Bool.parser()
 }
-.parse("123true")
+
+try parser.parse("123true")
 ```
 
-To fix this you can force one of the integer parsers to be the `Substring` parser, and then the
+To fix this you can force one of the parsers to be the `Substring` parser, and then the
 other will figure it out via type inference:
 
 ```swift
-try Parse {
-  Int.parser(of: Substring.self)
-  Bool.parser()
+let parser = Parse {
+  Int.parser // ✅
+  Bool.parser(of: Substring.self)
 }
```

Comment:
When `Int.parser()` and `Bool.parser()` are called in one block, type inference will fail.

Compiler Error:
`Type '(Substring.UTF8View.Type, Bool, Int) -> Parsers.IntParser<Substring.UTF8View, Int>' cannot conform to 'Parser'`